gatsby-config.js

```diff
@@ -13,29 +13,45 @@ const queries = require("./src/utils/algolia");
 const getPlugins = () => {
   const plugins = [
     {
-      resolve: "@sentry/gatsby",
+      resolve: '@sentry/gatsby',
       options: {
         dsn:
           process.env.SENTRY_DSN ||
-          "https://f107f3f0deb544289e4e056922e5e5a4@o1.ingest.sentry.io/5266138",
-        tracesSampleRate: activeEnv === "development" ? 0 : 1,
+          'https://f107f3f0deb544289e4e056922e5e5a4@o1.ingest.sentry.io/5266138',
+        tracesSampleRate: activeEnv === 'development' ? 0 : 1,
       },
     },
-    "gatsby-plugin-sass",
-    "gatsby-plugin-sharp",
-    "gatsby-plugin-zeit-now",
+    'gatsby-plugin-sass',
+    'gatsby-plugin-sharp',
+    'gatsby-plugin-zeit-now',
     {
-      resolve: "gatsby-plugin-mdx",
+      resolve: 'gatsby-plugin-google-gtag',
       options: {
-        remarkPlugins: [require("remark-deflist")],
+        // You can add multiple tracking ids and a pageview event will be fired for all of them.
+        trackingIds: [
+          'UA-30327640-1', // Sentry
+          'UA-30327640-3', // www
```

Comment:
do we actually want www on here? not really sure

Reply:
This was what was done earlier in the docs repo. I'm not a fan of duplication so I'll remove it.